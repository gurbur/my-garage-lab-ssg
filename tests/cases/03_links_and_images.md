## 링크 종합 테스트

이 문단에는 여러 종류의 링크가 있습니다.

표준 마크다운 링크는 이렇게 생겼습니다: [Google](https://google.com).
Obsidian 내부 글 링크는 이렇게 생겼습니다: [[(1) 첫 글]].
마지막으로, Obsidian 이미지 링크는 이렇게 생겼습니다:

![[image_preview.png]]

링크들 사이에도 일반 텍스트가 올 수 있습니다.
