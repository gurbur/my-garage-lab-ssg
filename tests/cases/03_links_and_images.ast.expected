-> DOCUMENT
	-> HEADING2 | data1: "링크 종합 테스트"
	-> PARAGRAPH
		-> TEXT | data1: "이 문단에는 여러 종류의 링크가 있습니다."
	-> PARAGRAPH
		-> TEXT | data1: "표준 마크다운 링크는 이렇게 생겼습니다: "
		-> LINK | data1: "Google" | data2: "https://google.com"
		-> TEXT | data1: "."
		-> SOFT_BREAK
		-> TEXT | data1: "Obsidian 내부 글 링크는 이렇게 생겼습니다: "
		-> LINK | data1: "(1) 첫 글" | data2: "/(1) 첫 글"
		-> TEXT | data1: "."
		-> SOFT_BREAK
		-> TEXT | data1: "마지막으로, Obsidian 이미지 링크는 이렇게 생겼습니다:"
	-> PARAGRAPH
		-> IMAGE_LINK | data1: "image_preview.png" | data2: "/tests/mock_vault/assets/image_preview.png"
	-> PARAGRAPH
		-> TEXT | data1: "링크들 사이에도 일반 텍스트가 올 수 있습니다."
		-> SOFT_BREAK
